release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1384,6 +1374,15 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 |General Availability
 
+|{product-title} on Oracle Cloud@Customer (C3)
```

Comment:
"Cloud@Customer" is the official name? And "C3" refers to "Cloud", "Customer", and something else?

Reply:
Oracle Compute Cloud@Customer is the name of the service. We need to keep (C3) at the end as this is how Oracle and their customers refer to this service.

https://www.oracle.com/uk/cloud/compute/cloud-at-customer/

Reply:
Thanks, Marcos, so the name just needs to be updated to include "Compute".